### PR TITLE
TDLR-2526 - replace EADs with reference to ASpace Finding Aid in Coll…

### PR DESCRIPTION
**Pull request by mkorcy**
…ections.

### CODE REVIEW

File: lib/tufts/curation/file_set_indexer.rb [hunk 1]
```diff
@@ -33,29 +29,6 @@ def generate_solr_document
...
-              end
-            end
-            # Only do this after the indexer has the file_set
-            load_ead_xml(object)
-            if @noko.nil?
-              Rails.logger.warn("Couldn't find the Ead XML for #{solr_doc['id']}")
-            else
-              solr_doc['all_text_timv'] = @noko.xpath('//text()').text.gsub(/[^0-9A-Za-z]/, ' ')
-            end
-          end # end if
```
Comment (mkorcy) [inline lib/tufts/curation/file_set_indexer.rb:58]: I'm not sure if this is right, I realize it says #ead customizations but, do we just want to remove lines 52-57 or this whole block?

File: lib/tufts/curation/vocab/tufts.rb [hunk 2]
```diff
@@ -8,7 +8,6 @@ module Vocab
       # http://dl.tufts.edu/terms/identifier
       class Tufts < RDF::Vocabulary('http://dl.tufts.edu/terms#')
         # term :VotingRecord
-        # term :EAD
```
Comment (mkorcy) [inline lib/tufts/curation/vocab/tufts.rb:11]: There's no data you need to clear out of this field before you make it inaccessible?

**Review by mkorcy — COMMENTED** [review]
@bgoodmon I had 2 questions on this.

File: lib/tufts/curation/file_set_indexer.rb (hunk 1 above)
Comment (bgoodmon) [reply lib/tufts/curation/file_set_indexer.rb:58]: I think you're right - although I'm not totally sure how that method worked for non-EAD items.  I put it back minus the load_ead_xml(object) call and if @noko.nil? block.

File: lib/tufts/curation/vocab/tufts.rb (hunk 2 above)
Comment (bgoodmon) [reply lib/tufts/curation/vocab/tufts.rb:11]: Ah, you're right, it might be handy for the data remediation phase.  I'll put it back.

Comment (bgoodmon) [reply lib/tufts/curation/vocab/tufts.rb:11]: What does it mean that it was commented out?

**Review by mkorcy — COMMENTED** [review]
@bgoodmon this all looks good, i can't approve because i created the PR but its good to go.

**Pull request merged**